tests/tests_integration/test_api/test_functions.py

```diff
@@ -198,3 +198,20 @@ def test_raise_retrieve_unknown(self, cognite_client: CogniteClient) -> None:
         with pytest.raises(CogniteNotFoundError):
             cognite_client.functions.schedules.retrieve(id=[123])
         assert cognite_client.functions.schedules.retrieve(id=123) is None
+
+    def test_create_with_nonce(self, cognite_client: CogniteClient, dummy_function: Function) -> None:
+        session = cognite_client.iam.sessions.create(session_type="ONESHOT_TOKEN_EXCHANGE")
+        schedule = FunctionScheduleWrite(
+            name="test_create_with_nonce",
+            cron_expression="0 0 * * *",
+            function_id=dummy_function.id,
+            data={"key": "value"},
+        )
+        created: FunctionSchedule | None = None
+        try:
+            created = cognite_client.functions.schedules.create(schedule, nonce=session.nonce)
```

Comment:
Can we add a check to make sure the nonce was actually used or is it part of the dumped repsonse?